src/screens/FeedScreen/index.js

```diff
@@ -315,6 +315,8 @@ const FeedScreen = (props) => {
     );
   };
 
+  const contentHeight = dimen.size.FEED_CURRENT_ITEM_HEIGHT;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable `contentHeight` is declared but it's not clear where the value for `dimen.size.FEED_CURRENT_ITEM_HEIGHT` comes from. If this value is constant, it would be better to declare it at the top of the file or import it from a constants file. This will improve readability and maintainability.
```diff
-  const contentHeight = dimen.size.FEED_CURRENT_ITEM_HEIGHT;
+  const FEED_CURRENT_ITEM_HEIGHT = someConstantValue; // replace with actual value
+  const contentHeight = FEED_CURRENT_ITEM_HEIGHT;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->